python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -56,6 +66,14 @@ def df_pair(self):
         psdf = ps.from_pandas(pdf)
         return pdf, psdf
 
+    def test_validate_test_names(self):
+        # All test cases MUST be in pattern 'test_[a-z].*',
```

Comment:
What if the test name started with upper-case? e.g. `test_Aggregate` ? (Of course, this is also a basic violation of Python naming conventions)

Reply:
yes, right now we don't have such a method.
I added an extra method `test_validate_test_names` to detect if there is some function violate such pattern